### PR TITLE
rename the ESQuerySet used in the api module to ElasticAPIQuerySet

### PR DESCRIPTION
**Pull request by czue**
this was pretty confusing since there's also [this `ESQuerySet`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/es_query.py#L331)

@esoergel @snopoke @benrudolph 

wait for tests

### CODE REVIEW

File: corehq/apps/api/resources/v0_4.py [hunk 1]
```diff
@@ -96,9 +96,11 @@ def wrapper(doc):
...
 
         # Note that XFormES is used only as an ES client, for `run_query` against the proper index
-        return ESQuerySet(payload=es_query,
-                          model=wrapper,
-                          es_client=self.xform_es(domain)).order_by('-received_on')
+        return ElasticAPIQuerySet(
+            payload=es_query,
+            model=wrapper,
+            es_client=self.xform_es(domain)).order_by('-received_on'
+        )
```
Comment (dimagimon) [inline corehq/apps/api/resources/v0_4.py]: E124 closing bracket does not match visual indentation

File: corehq/apps/api/resources/v0_4.py [hunk 2]
```diff
@@ -426,9 +430,11 @@ def obj_get_list(self, bundle, domain, **kwargs):
...
 
         # Note that CaseES is used only as an ES client, for `run_query` against the proper index
-        return ESQuerySet(payload=query,
-                          model=HOPECase,
-                          es_client=self.case_es(domain)).order_by('server_modified_on')
+        return ElasticAPIQuerySet(
+            payload=query,
+            model=HOPECase,
+            es_client=self.case_es(domain)).order_by('server_modified_on',
+        )
```
Comment (dimagimon) [inline corehq/apps/api/resources/v0_4.py]: E124 closing bracket does not match visual indentation

File: corehq/apps/api/tests.py [hunk 3]
```diff
@@ -32,7 +32,7 @@
 from corehq.apps.api.resources import v0_1, v0_4, v0_5
 from corehq.apps.api.fields import ToManyDocumentsField, ToOneDocumentField, UseIfRequested, ToManyDictField
 from corehq.apps.api import es
-from corehq.apps.api.es import ESQuerySet, ESUserError
+from corehq.apps.api.es import ElasticAPIQuerySet, ESUserError
```
Comment (dimagimon) [inline corehq/apps/api/tests.py]: F401 'ESUserError' imported but unused

File: corehq/apps/api/tests.py [hunk 4]
```diff
@@ -762,31 +761,31 @@ def test_xformPillowTransform(self):
...
     '''
-    Tests the ESQuerySet for appropriate slicing, etc
+    Tests the ElasticAPIQuerySet for appropriate slicing, etc
     '''
 
     def test_slice(self):
         es = FakeXFormES()
         for i in xrange(0, 1300):
-            es.add_doc(i, {'i': i})
+            es.add_doc(i, {'i': i})<
```
Comment (dimagimon) [inline corehq/apps/api/tests.py]: E225 missing whitespace around operator

Comment (dimagimon) [inline corehq/apps/api/tests.py]: E901 SyntaxError: invalid syntax

**Comment by czue** [comment]
:white_check_mark:

**Comment by esoergel** [comment]
:+1:

**Pull request merged**